### PR TITLE
docs: Update Kubernetes operator guide

### PR DESCRIPTION
**Pull request by ericgregory**
Integrate updates to wasmcloud-operator quickstart.

### CODE REVIEW

File: docs/deployment/k8s/index.mdx [hunk 1]
```diff
@@ -37,172 +37,123 @@ If you are running Kubernetes locally, the following steps will help you start a
...
+```shell
+helm repo add nats https://nats-io.github.io/k8s/helm/charts/
 ```
-
-Run `helm install`:
+Install the upstream NATS Helm chart to start a cluster with the `values.yaml` file from the quickstart folder:
 
 ```shell
-helm install wadm -f values.yaml oci://ghcr.io/wasmcloud/charts/wadm
+helm upgrade --install -f nats-values.yaml nats nats/nats
```
Comment (joonas) [inline docs/deployment/k8s/index.mdx]: In order to no longer having to git checkout the operator repository, we can instead reference the quick start values files via URL:
```suggestion
helm upgrade --install -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/nats-values.yaml nats nats/nats
```

File: docs/deployment/k8s/index.mdx [hunk 2]
```diff
@@ -37,172 +37,123 @@ If you are running Kubernetes locally, the following steps will help you start a
...
 
-The `WasmCloudHostConfig` CRD describes the desired state of a set of wasmCloud hosts connected to the same lattice. We will use `wash` to generate a public key and seed for the new wasmCloud hosts that will run on our Kubernetes cluster:
+You can deploy wadm to your Kubernetes cluster with a Helm chart hosted by wasmCloud. 
 
 ```shell
-wash keys gen cluster
-
-Public Key: <your key>
-Seed: <your seed>
+helm install wadm -f wadm-values.yaml oci://ghcr.io/wasmcloud/charts/wadm
```
Comment (joonas) [inline docs/deployment/k8s/index.mdx]: Same as NATS above:
```suggestion
helm install wadm -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/wadm-values.yaml oci://ghcr.io/wasmcloud/charts/wadm
```

File: docs/deployment/k8s/index.mdx [hunk 3]
```diff
@@ -37,172 +37,123 @@ If you are running Kubernetes locally, the following steps will help you start a
...
-  secretName: my-wasmcloud-cluster
-  # Enable the following to run the wasmCloud hosts as a DaemonSet
-  #daemonset: true
-  # The name of the image pull secret to use with wasmCloud hosts so that they
-  # can authenticate to a private registry to pull components.
-  # registryCredentialsSecret: my-registry-secret
+To deploy the operator:
+
+```shell
+kubectl apply -k ../../deploy/base
```
Comment (joonas) [inline docs/deployment/k8s/index.mdx]: Same as above:
```suggestion
kubectl apply -k https://github.com/wasmCloud/wasmcloud-operator/deploy/base
```

File: docs/deployment/k8s/index.mdx [hunk 4]
```diff
@@ -37,172 +37,123 @@ If you are running Kubernetes locally, the following steps will help you start a
...
-metadata:
-  name: my-wasmcloud-cluster
-stringData:
-  # You can generate this with wash:
-  # `wash keys gen cluster`
-  WASMCLOUD_CLUSTER_SEED: <seed>
-```
 ```shell
-kubectl apply -f secret.yaml
+kubectl apply -f wasmcloud-host.yaml
```
Comment (joonas) [inline docs/deployment/k8s/index.mdx]: Same as above:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/wasmcloud-host.yaml
```

File: docs/deployment/k8s/index.mdx [hunk 5]
```diff
@@ -212,34 +163,36 @@ spec:
             source_config:
               - name: default-http
                 properties:
-                  ADDRESS: 127.0.0.1:8080
-
+                  address: 127.0.0.1:8080
```
Comment (joonas) [inline docs/deployment/k8s/index.mdx]: Running this on `127.0.0.1` will be problematic inside Kubernetes, let's use `0.0.0.0` instead:
```suggestion
                  address: 0.0.0.0:8080
```

File: docs/deployment/k8s/index.mdx [hunk 6]
```diff
@@ -37,172 +37,123 @@ If you are running Kubernetes locally, the following steps will help you start a
...
-      domain: default
-  websocket:
-    enabled: true
-    port: 4223
-```
-
 ```shell
-helm upgrade --install -f values.yaml nats-cluster nats/nats
+cd examples/quickstart
 ```
```
Comment (joonas) [inline docs/deployment/k8s/index.mdx]: With the changes below, the `Download the operator` section is not needed.

Comment (ericgregory) [reply docs/deployment/k8s/index.mdx]: Removed in the latest commit!

**Review by joonas — APPROVED** [review]

**Review by lachieh — APPROVED** [review]
:stamped:

**Pull request merged**